lib/api/apiUtils/object/versioning.js

```diff
@@ -212,35 +212,29 @@ function getMasterState(objMD) {
  */
 function versioningPreprocessing(bucketName, bucketMD, objectKey, objMD,
```

Comment:
same feedback as above

Reply:
This function has jsdoc already, not sure if you are referring to something more specific